integration-cli/requirements.go

```diff
@@ -153,6 +153,19 @@ var (
 		},
 		"Test requires support for IPv6",
 	}
+	UserNamespaceROMount = testRequirement{
+		func() bool {
+			// quick case--userns not enabled in this test run
+			if os.Getenv("DOCKER_REMAP_ROOT") == "" {
+				return true
+			}
+			if _, _, err := dockerCmdWithError("run", "--read-only", "busybox", "date"); err != nil {
```

Comment:
I'm guessing you also need to run this with `--rm` or make sure its cleaned up correctly?